provider/ansible/property_override.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Google Inc.
```

Comment:
Add in the issue title that you are also adding it for ansible.

Reply:
Done.